Gemfile

```diff
@@ -6,6 +6,8 @@ gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
 
 group :development do
   gem 'rspec'
+  gem 'coveralls'
+  gem 'simplecov'  
```

Comment:
Its better to add development dependencies that do not require a dynamic references to guthub, etc, to gemspec.